diff --git src/webrtc.gni src/webrtc.gni
index 7154b8760..1dceed09c 100644
--- src/webrtc.gni
+++ src/webrtc.gni
@@ -14,20 +14,6 @@ import("//build/config/ui.gni")
 import("//build_overrides/build.gni")
 import("//testing/test.gni")
 
-if (!build_with_chromium && is_component_build) {
-  print("The Gn argument `is_component_build` is currently " +
-        "ignored for WebRTC builds.")
-  print("Component builds are supported by Chromium and the argument " +
-        "`is_component_build` makes it possible to create shared libraries " +
-        "instead of static libraries.")
-  print("If an app depends on WebRTC it makes sense to just depend on the " +
-        "WebRTC static library, so there is no difference between " +
-        "`is_component_build=true` and `is_component_build=false`.")
-  print(
-      "More info about component builds at: " + "https://chromium.googlesource.com/chromium/src/+/master/docs/component_build.md")
-  assert(!is_component_build, "Component builds are not supported in WebRTC.")
-}
-
 if (is_ios) {
   import("//build/config/ios/rules.gni")
 }
